index.js

```diff
@@ -2,12 +2,14 @@ const { exec } = require('child_process');
 const loaderUtils = require('loader-utils');
 
 module.exports = function(source) {
-    var callback = this.async();
-    var options = loaderUtils.getOptions(this);
-    var command = exec(options.script, function(err, result) {
+    const { script, ...options } = loaderUtils.getOptions(this);
+    const callback = this.async();
```

Comment:
Just a small nitpick: can you put this line back at the top of the function and remove the extra whitespace below and on line 12?

Reply:
sure, it's done!